### ISSUE TITLE
Error when trying to update 1.6.3 (64) to 1.6.3 (65) on macOS High Sierra

### ISSUE THREAD

**Issue by tempelmann** [samschott/maestral#778]
I am running 1.6.3 (64) on High Sierra. When I check for updates, it offers build 65.

When I choose to install, it downloads the update, but then says:

    Update Error!
    An error occurred while extracting ...

I believe it's not about the extraction but about the fact that the new build may require 10.15 when it should still work on 10.13.

I'd expect that as long as the version remains the same, that bug fixes keep working on the same macOS version. (I know that 1.6.4 does not work on H.S.)

As a general rule: When you change the fundamental behavior of a program, including new requirements, the version number should be increased at a higher level than the bugfix (x.x.bugfix) level. Ideally, the top level should be incremented, but at least turn a 1.6.3 into a 1.7 for that next time.

**Comment by samschott** [comment]
Fair point, the next such "major" change will get an appropriate version bump.

I'm surprised the update was offered to you at all, Sparkle should not have notified in this case. I'll look into that. Unfortunately, any fix that goes beyond changing the app cast data available server-side won't reach you anymore on High Sierra :(
